packages/ses-ava/test/prepare-test-env-ava.js

```diff
@@ -0,0 +1,7 @@
+import '@endo/lockdown/commit-debug.js';
```

Comment:
Besides migrating, also changed this from `'@endo/init/debug.js'` to avoid depending on @endo/eventual-send